src/main/java/us/kbase/auth2/lib/UserSearchSpec.java

```diff
@@ -353,10 +372,50 @@ public Builder withIncludeDisabled(final boolean include) {
 		
 		/** Build a UserSearchSpec instance.
 		 * @return a UserSearchSpec.
+		 * @throws IllegalParameterException if a prefix is set that, after normalizing, contains
+		 * no characters for the requested search(es).
 		 */
-		public UserSearchSpec build() {
-			return new UserSearchSpec(prefixes, regex, searchUser, searchDisplayName, searchRoles,
-					searchCustomRoles, includeRoot, includeDisabled);
+		public UserSearchSpec build() throws IllegalParameterException {
+			List<String> userNamePrefixes = null;
+			List<String> displayPrefixes = null;
+			if (this.prefix != null) {
+				/*	UsrSrch	DisSrch	UsrOK	DisOK	Throw exception?
+				 *	T		T		Y implies Y		
```

Comment:
The formatting of this is messed up - looks like your tabs are set to 16 spaces and GitHub uses 8.

Reply:
My tabs are set to 4, github's default is 8. If you set github to 4 it looks normal (top right profile button -> settings -> appearance -> tab size preference)